test_app/after/src/inline_items.erl

```diff
@@ -47,7 +47,10 @@ short_list() ->
     [X, Y, [x | y:z()]].
 
 -spec short_fun() -> fun((X, Y, Z) -> {X, Y, Z}).
-short_fun() -> fun (X, Y, Z) -> {X, Y, Z} end.
+short_fun() ->
+    fun (X, Y, Z) ->
+            {X, Y, Z}
```

Comment:
8 indentation spaces here between `fun` and its body, is that ok?

Reply:
maybe not… but with just 4 it will look like this…
```erlang
    fun (X, Y, Z) ->
        {X, Y, Z}
    end
```
…which doesn't look _that_ bad until you add another clause…
```erlang
    fun (X, y, Z) ->
        {X, y, Z};
        (X, w, Z) ->
        {X, w, Z}
    end
```